carp.common/src/commonMain/kotlin/dk/cachet/carp/common/application/users/ExpectedParticipantData.kt

```diff
@@ -0,0 +1,66 @@
+package dk.cachet.carp.common.application.users
+
+import dk.cachet.carp.common.application.data.input.InputDataType
+import kotlinx.serialization.Serializable
+
+
+/**
+ * Describes a participant [attribute] [assignedTo] to be input by specified users in a study.
+ */
+@Serializable
+data class ExpectedParticipantData(
+    val attribute: ParticipantAttribute,
+    val assignedTo: AssignedTo = AssignedTo.Anyone
+)
+{
+    val inputDataType: InputDataType
+        get() = attribute.inputDataType
+}
+
+
+/**
+ * Determines whether the set contains any of the following conflicts:
+ * - contains differing participant attributes with the same input type
```

Comment:
I think it should be clarified what 'type' means i.e. that it's more than just, say, a free-form text input.